### DISCUSSION TITLE
Artifact-hub chart in air-gap env

### DISCUSSION THREAD

**Discussion by zerr0s** [artifacthub/hub#3945] [Q&A] [answered]
Hello,

I have a probel to deploy artifact-hub in an air-gap environment.
The schema validation is blocking the deployment because the "$ref" url can not be reached (normal in my case).

Is there a way to fix it ? Should I fork the official repo in my local gitlab and change all urls ?

Thanks.

**Comment by tegioz** [comment, marked as answer]
Hi!

The following PR provides a solution for this problem (still not merged though):

https://github.com/helm/helm/pull/12743 (and https://github.com/helm/helm-www/pull/1549)

In the meantime, you could just download the chart and remove or rename the `values.schema.json` file to disable the schema validation. Replacing the urls of all references in the schema sounds good. Maybe you can even do it directly on the schema file after downloading the chart (i.e. using `sed` or something similar).

**Reply by zerr0s** [reply]
Ok thanks. I will bypass the schema validation for now. Hope that the PR will me merged soon.
